src/video/src/Context_PSP.cpp

```diff
@@ -55,7 +55,6 @@ bool Video::setup_gl()
         return false;
     }
 
-    SDL_GL_LoadLibrary(nullptr);
```

Comment:
This turned out to be completely unnecessary, according to [the docs](https://wiki.libsdl.org/SDL_GL_LoadLibrary#Remarks):
```
This should be done after initializing the video driver, but before creating any OpenGL windows. If no OpenGL library is loaded, the default library will be loaded upon creation of the first OpenGL window. 
```

Reply:
The docs you linked refer to SDL2.

Nevertheless, it is not needed.
https://www.libsdl.org/release/SDL-1.2.15/docs/html/sdlglloadlibrary.html

Reply:
My bad, sorry, but that's what I meant.